config/opensearch_dashboards.yml

```diff
@@ -183,3 +183,10 @@
 # Set the value of this setting to false to suppress search usage telemetry
 # for reducing the load of OpenSearch cluster.
 # data.search.usageTelemetry.enabled: false
+
+# Set the value of this setting to true to enable credential management for managing 
+# the credentials of OpenSearch cluster.
+# It's enabled now for for playground env CD
+# TODO: Disable it for release
+credential_management.enabled: true
```

Comment:
we don't need to add it to this yml file, just set the default value in the plugin ts file

Reply:
And default should be false

Reply:
For playground  CD purpose, do we need to ssh in to the ec2 instance and change the plugin file to enable CM?

Reply:
irrelevant to playground. let's target production